style.less

```diff
@@ -0,0 +1,93 @@
+span.qc_icon {
+    line-height: 1.2em;
+    display: inline-block;
+    margin-left: 0.5em;
+    svg {
+        width: 1.2em;
+        height: 1.2em;
+        vertical-align: top;
+    }
+
+    &.qc_bad {
+        fill: #933;
+        color: #933;
+    }
+
+    &.qc_meh {
+        fill: #993;
+        color: #993;
+    }
+
+    &.qc_good {
+        fill: #393;
+        color: #393;
+    }
+}
+
+
+div#plugin__qc__wrapper {
+    background-color: @ini_background_neu;
+    color: @ini_text_neu;
```

Comment:
hmm. AFAIK those colors are meant to be used in combination. So if the contrast is shitty, we should probably fix it in the default template. But yeah, styles could be optimized I guess.